pylammpsmpi/lammps.py

```diff
@@ -496,11 +499,14 @@ def command(self, cmd):
         if isinstance(cmd, list):
             for c in cmd:
                 self._send(command="command", data=c)
+                a = self._receive()
         elif len(cmd.split('\n')) > 1:
             for c in cmd.split('\n'):
                 self._send(command="command", data=c)
+                a = self._receive()
         else:
             self._send(command="command", data=cmd)
+            a = self._receive()
```

Comment:
I guess instead of using `a = self._receive()` the pythonic way would be `_ = self._receive()` but that is something we can change in the future.

Reply:
Commit e3c33a8d58efa1bebd9fcef1ad13f3b08f001690 fixes this. thanks for the feedback!